Addressing workshop feedback

artberger opened the pull request:
Addresses issue called out in slack channel: https://ibm-argonauts.slack.com/messages/CBKP5QLN9

Addresses #47, #48

jzaccone commented on workshop/deploy_rollup.sh:
```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
 echo "Install IBM Cloud CLI"
-. workshop/install_bx.sh
+. workshop/install_ibmcloud.sh
```
Need to change filename to match

MHBauer commented:
@artberger what was feedback? I can't see that slack.

MHBauer commented on workshop/Lab3/README.md:
````diff
@@ -100,7 +100,7 @@ try to add, or remove, pods from the system to match your request. To can
 
    ```console
-   $ kubectl edit deployment guestbook
+   $ kubectl edit deployment guestbook-v1
````
why would the deployment be called this?

MHBauer replied:
I see, guestbook has been updated. We need a better way to keep this in sync if we're going to have separate repos. Checking out a specific revision/branch or something.

artberger replied:
Yeah it had to do with the tag. Is there something you want me to change in this PR?

artberger commented:
@MHBauer this commit addressed the feedback in slack: https://github.com/IBM/kube101/pull/50/commits/ed9df25c482bc7501a1e582e9c4596bf727a9ae2, besides the plugin name change and no need for the alias.

duglin commented:
@brahmaroutu please review

MHBauer commented:
we need to rebase and merge this. I don't have perms to do either.

MHBauer commented:
I have some perms now, maybe @jzaccone can follow up.

stevemar commented:
Closing due to inactivity and conflicts
